tests/unit/test__signing.py

```diff
@@ -728,6 +741,7 @@ def _run_conformance_test(resource, test_data):
         method=test_data["method"],
         _request_timestamp=test_data["timestamp"],
         headers=test_data.get("headers"),
+        query_parameters=test_data.get("queryParameters"),
     )
 
```

Comment:
Please add a testcase class which explicitly tests the new `_quote_param` function with all kinds of expected values:

- Simple, non-quoted ASCII characters
- Characters which *must* be quoted (e.g., `"/"`)
- Characters which must *not* be quoted (`"~"`)
- Unicode characters, which must be converted to UTF-8 and encoded

The tests should assert that the returned value is of the correct type, as well as matching the expected value.